content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -87,7 +87,7 @@ The three conditions are `ready`, `serving`, and `terminating`.
 
 #### Ready
 
-`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
+`ready` is a condition that maps to a Pod's `ready` condition. A running Pod with the `Ready`
```

Comment:
This line has been revised several times and it is still wrong.
For a Pod, the condition is named `Ready`. However, for an endpoint, the condition is named `ready`.
So the first "ready" should be in `ready` and the second one should be `Ready`.
This is an inconsistency introduced by the upstream implementation, thus something we have to live with.

Can we get this revision done right now?


```suggestion
`ready` is a condition that maps to a Pod's `Ready` condition. A running Pod with the `Ready`
```

Reply:
@tengqm I have made the revision. Please have a look.